src/coreclr/tests/runtest.cmd

```diff
@@ -16,7 +16,7 @@ set "__ProjectDir=%~dp0"
 :: remove trailing slash
 if %__ProjectDir:~-1%==\ set "__ProjectDir=%__ProjectDir:~0,-1%"
 set "__ProjectFilesDir=%__ProjectDir%"
-set "__RootBinDir=%__ProjectDir%\..\..\..\artifacts"
+set "__RootBinDir=%~dp0..\..\..\artifacts"
```

Comment:
A nit - maybe use `__ThisScriptDir` instead of `%~dp0` here and in the change below.